active-rfcs/0001-contribution.md

```diff
@@ -0,0 +1,60 @@
+- Feature Name: Contribution
+- Start Date: 2021-03-01
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+A contribution plugin will be developed for tichi, which is mainly responsible for labeling external members' PRs with `contribution` or `first-time-contributor` labels.
+
+# Motivation
+
+We need to distinguish between internal and external contributor PRs, and prioritize help with external contributor PRs, especially first-time contributor PRs, which will allow external contributors to have a good contribution experience.
+
+# Detailed design
+
+## contribution
+
+For external contributors, we will label their PR with `contribution`.
+
+### Definition:
+
+- Label Name: `contribution`
+- Usage: External Contributors
+  - We will determine if it is an external contributor by whether it is an org member or not.
+- Available API: https://github.com/kubernetes/test-infra/blob/9b84ac948f6eb781a318fa431d547e577ee668e4/prow/github/client.go#L67
+
+## first-time-contributor
+
+In addition, if the external contributor is a first-time contributor, we need to tag not only `contribution` but also `first-time-contributor`.
+
+### Definition:
+
+- Label Name: `first-time-contributor`
```

Comment:
No need to distinguish between external and internal author in this case?

Reply:
I think it may may not be necessary, because the contribution will also be added according to whether it is members, we only need to distinguish whether he is the first contribution.